arbitrum-docs/partials/_troubleshooting-nodes-partial.md

```diff
@@ -68,15 +68,42 @@
 
 
 ### How can I verify that my node is fully synced? {#how-can-i-verify-that-my-node-is-fully-synced}
-<p>You can make an <code>eth_syncing</code> RPC call to your node. If it returns false, it means that the node is fully synced. If it returns an object of information, it means it is still syncing. Here's how you can interpret that information:</p>
+<p>You can make an <code>eth_syncing</code> RPC call to your node. When a nitro node is fully synced, <code>eth_syncing</code> returns the value <code>false</code> (just like a normal Geth node).</p>
+
+<p>When a nitro node is still syncing, <code>eth_syncing</code> returns a map of values to help understand why the node is not synced. Nitro execution and bottleneck are different from a normal Geth node, so <code>eth_syncing</code> output is unique to nitro. Here, we provide more details:</p>
+
+<p></p>
+
+<p><strong>Introduction: Messages, Batches, and Blocks:</strong></p>
+
+<p>Nitro node reads <strong>messages </strong>from the parent chain an optionally from the message feed. It executes these messages and produces <strong>blocks.</strong> Each message produces exactly one block (a message may contain multiple transactions). In most nitro chains, message number and block number are the same. However, Arbitrum One chain has pre-nitro blocks, so for that chain message 0 produced block #XXXX, the offset between message and block number is constant in the chain.</p>
+
+<p>On the parent chain, messages appear in batches. The number of messages per batch changes between batches.</p>
 
-<ul><li><code>{'lastl1blockNum'}</code> specifies what the node sees as the current L1 state</li>
-<li><code>{'blockNum'}</code> is what's the up-to-date L2 block the node is synced to</li>
-<li><code>{'messageOfLastBlock'}</code> is the message number of that block the node is synced to</li>
-<li><code>{'broadcasterQueuedMessagesPos'}</code> specifies the last message number that the feed has broadcasted</li>
-</ul>
 <p></p>
 
+<p><strong><code>{'eth_syncing'}</code></strong><strong> Fields:</strong></p>
+
+<p>🔉 Note that the exact output for <code>eth_syncing</code> RPC call of an out-of-sync nitro node is not considered an stable API. It is still being actively developed and modified without notice between versions.</p>
```

Comment:
Not sure it is possible, but I would use a caution notice for this.
Like the ones used in the [Quickstart](https://developer.arbitrum.io/for-devs/quickstart-solidity-hardhat), (search for Caution)